.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -0,0 +1,78 @@
+---
+name: 🐛 Bug Report
+about: If something isn't working as expected 🤔.
+
+---
+
+<!---
+Before you publish a bug report, please make sure you have have made a reasonable effort to eliminate any outside causes.
+
+* Check the official documentation for the resource that errors to see whether you have configured it incorrectly, or it's being used incorrectly within the module.
+* Make sure you have the latest stable version of your cloud provider version and see if anything has change (which is related to the resource that errors) in newer releases of the provider.
+* Made sure you have performed validation (`terraform fmt`, `terraform validate` or `tflint`) against your terraform workspace.
+* Searched for any changes or issues regarding the terraform provider in question (eg. https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md)
+
+If you are running into one of these scenarios, we recommend you reaching out to the terraform community, see information further down.
+--->
+
+<!--- Please keep this note for the community --->
+
+### Community Note
+
+* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
+* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
+* If you are interested in working on this issue or have submitted a pull request, please leave a comment
+
+<!--- Thank you for keeping this note for the community --->
+
+### Terraform Version
+
+<!--- Please run `terraform -v` to show the Terraform core version and provider version(s). If you are not running the latest version of Terraform or the provider, please upgrade because your issue may have already been fixed. [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions). --->
+
+### Affected Resource(s)
+
+<!--- Please list the affected resources and data sources. --->
+
+* Resources
+
+### Terraform Configuration Files
+
+<!--- Information about code formatting: https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code --->
+
+```hcl
+# Copy-paste your Terraform configurations here - for large Terraform configs,
+# please use a service like Dropbox and share a link to the ZIP file. For
+# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
```

Comment:
To have this line we would need a public key they can encrypt it with. Also, is there any case where users should send us secret things?